test/e2erp/realrp_test.go

```diff
@@ -3,45 +3,61 @@
 package e2erp
 
 import (
-	"context"
+	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/sirupsen/logrus"
-
-	"github.com/openshift/openshift-azure/pkg/api"
 	"github.com/openshift/openshift-azure/pkg/log"
+	"github.com/sirupsen/logrus"
```

Comment:
https://github.com/openshift/openshift-azure/pull/764

Reply:
had to rework to get rid of logger stuff, using logrus.* now